requirements.txt

```diff
@@ -1,9 +1,9 @@
-coincurve>=8.0.0,<9.0
+coincurve>=9.0.0,<10.0
```

Comment:
why update it here and not in [raiden](https://github.com/raiden-network/raiden/blob/b1867299a8de793a4991ae560abc9898c6c6673b/constraints.txt#L11)? And why update dependencies at a feature freeze state of the code?

Reply:
It's simple logic actually.

If something broke in the dependency we may get a change in raiden behaviour or a nasty bug. I would avoid updating dependencies at this point unless there is a very good reason.

If it's not needed don't do it.